### DISCUSSION TITLE
win64 zip version is missing

### DISCUSSION THREAD

**Discussion by jackycms** [doublecmd/doublecmd#597] [General]
Hi

doublecmd-1.0.8.x86_64-win64.zip is missing on download page,
i use zip version only.

keep up your good work

**Comment by blackcrack** [comment]
what's about "show all assets"
and short over the both sourcecodes packages ..

Chinece.. ! please !! common ....
so dump can not be a Chinese alone !
i hate fake accounts !

**Reply by jackycms** [reply]
i found the file on https://sourceforge.net/p/doublecmd/wiki/Download/  after i ask this question ,

i've only one github account, this is not fake account.
